crates/net/network/src/transactions/mod.rs

```diff
@@ -1063,6 +1063,9 @@ where
 /// [`crate::NetworkManager`] for more context on the design pattern.
 ///
 /// This should be spawned or used as part of `tokio::select!`.
+//
+// spawned in `NodeConfig::start_network`(reth_node_core::NodeConfig) and
+// `NetworkConfig::start_network`(reth_network::NetworkConfig)
```

Comment:
we don't depend on the node-core crate here, so we can remove reference to this but keep the link to Networkconfig

Reply:
ok it wasn't clear since it's spawned for both